client/web/compose/src/components/PageBlocks/Report/Configurator.vue

```diff
@@ -132,6 +153,26 @@ export default {
     setDefaultValues () {
       this.reports = []
     },
+
+    reduceElement (o) {
```

Comment:
Maybe we should sync about this because one of the other reason why this is like this is so that I can pass the appropriate data type to `c-input-select`